From 5ca47c1114f6fd442535dcc1b06b2291fa621cf5 Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Fri, 16 Feb 2018 15:14:11 -0500
Subject: [PATCH 171/176] arm: dts: am57xx ptp boundary clock dt bindings

Added PTP boundary clock default dt bindings for AM571x and AM572x

Signed-off-by: WingMan Kwork <w-kwok2@ti.com>
---
 arch/arm/boot/dts/am57xx-idk-common.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/am57xx-idk-common.dtsi b/arch/arm/boot/dts/am57xx-idk-common.dtsi
index af9c6b0..04d1766 100644
--- a/arch/arm/boot/dts/am57xx-idk-common.dtsi
+++ b/arch/arm/boot/dts/am57xx-idk-common.dtsi
@@ -188,6 +188,11 @@
 			/*tx-queue-size = /bits/ 16 <97 97 97 97 48>;*/
 		};
 	};
+
+	ptp_bc: ptp_bc {
+		compatible = "ti,am57-bc";
+		status = "disabled";
+	};
 };
 
 &dra7_pmx_core {
-- 
1.9.1

